CeeFM/Decideify
# Beginning Work on Rendering Current User's Suggestions in Appropriate Places

**Description**
The title kinda says it all. This pull request sorts out some of the quirks and errors we were having with the suggestionrepository.cs file - specifically hadn't built out the getbyuserid method properly, and didn't find those issues until I had mistakenly been troubleshooting the frontend for the cause for this issue. Alas, the life of a full stack developer.

**Type of Change**


**Testing Instructions**

- Clone this project’s repository to your machine
  -If you’ve already done this and are currently working on a different branch of the project, be sure to `git add` and `git commit` whatever you’re working on on your branch, before you attempt to switch branches
  -Then `git checkout main` to switch to the project’s main branch)
- Navigate to the root directory for this repo (yourParentFolderName/Decideify)
- `git fetch -a` to retrieve the current remote-tracked branches for the project, you should see a branch named `cm-suggestions` as one of the available branches
- `git checkout cm-suggestions` to switch to my branch
- If you don't see the most recent version of my files, `git pull origin cm-suggestions`
- `Start Decideify.sln` to open the project in Visual Studio
- If you haven’t created the Decideify database, please do so in Visual Studio and use the following SQL scripts to populate it accordingly:
  - Start a new query in your database and copy in this SQL script before running to fill out tables and columns: [First SQL Script for Creating DB](https://github.com/CeeFM/Decideify/blob/main/SQL/Decideify_Create_DB.sql)
  - ctrl + a and delete what is in that query and replace it with this SQL script before running to seed the database: [Second SQL Script for Seeding DB](https://github.com/CeeFM/Decideify/blob/main/SQL/Decideify_Seed_Users_DB.sql)

- Click the execute with debugger solid green button to launch the app
- Oh wait there's not really anything to test in this pull request, it was all backend updates. You can go to swagger and check the /api/suggestion/{id} endpoint and enter a user's ID and get their saved suggestions returned back to you. If they don't have any, obviously it will be an empty return.

Checklist
